### ISSUE TITLE
Implement a method to test plots

### ISSUE THREAD

**Issue by Cadair** [sunpy/sunpy#410]
Tests that test plotting need to be handled better as Tox can not create a GUI window. 
```
TclError: no display name and no $DISPLAY environment variable
```

Perhaps we can learn from this:
http://matplotlib.org/devel/testing.html#writing-an-image-comparison-test

**Comment by Cadair** [comment]
See this discussion on astropy-dev about where to put reference images. https://groups.google.com/forum/#!topic/astropy-dev/WahIPJ1L_3c

**Comment by ayshih** [comment]
Closed by #1277
